lively.morphic/svg-morph/svg-morph.js

```diff
@@ -140,15 +152,107 @@ export class SVGMorph extends Morph {
     selection_node.attr({
       'fill-opacity': 0.0,
       stroke: '#000',
-      'stroke-width': 2,
-      'pointer-events': 'none'
+      'stroke-width': 1,
+      'pointer-events': 'none',
+      'stroke-dasharray': '5, 3'
     });
     tar.after(selection_node);
     selection_node.front();
   }
 
   get isSVGMorph () { return true; }
 
+  getControlPoints () {
+    this.removeAllControlPoints();
+
+    const tar = SVG(this.target);
+    const targetPath = SVG(this.target).array();
+
+    for (let i = 0; i < targetPath.length; i++) {
+      let element = targetPath[i];
+      switch (element[0]) {
```

Comment:
Why a switch case here?

Reply:
this is old and would be a possibility to e.g. change color if we find certain line commands... but I can change it

Reply:
I don´t see a use case where we would need different colors for different lines, but maybe I am missing something